source/extensions/filters/http/jwt_authn/BUILD

```diff
@@ -69,6 +69,7 @@ envoy_cc_extension(
     srcs = ["filter_factory.cc"],
     hdrs = ["filter_factory.h"],
     security_posture = "robust_to_untrusted_downstream",
+    status = "alpha",
```

Comment:
I think this is used in production now? @lizan @qiwzhang? If so and this is still using alpha protos can we promote to non-alpha?